src/gpb_names.erl

```diff
@@ -488,15 +488,15 @@ enum_renamings(PkgByProto, PkgRenamings, Defs, RenameOps) ->
          end
          || {{enum_containment, Proto}, EnumNames} <- Defs])).
 
-enum_field_renamings(PkgByProto, Defs, RenameOps) ->
+enum_field_renamings(_PkgByProto, Defs, RenameOps) ->
     dict:from_list(
       lists:append(
         [begin
              [begin
               NewFieldName = run_ops(enum_fieldname, FieldName, Proto, RenameOps),
               {FieldName, NewFieldName}
               end
-              || {FieldName, _Value} <- EnumFields]
+              || {FieldName, _Value, _Options} <- EnumFields]
```

Comment:
Question got lost but was: for this and the change in `rename_enum_fields`, is there now always an options field/do we need to consider backwards compatibility at all?

Reply:
There should always be an options field in how this library now - my understanding is that this code is just for parsing and compiling the proto files, so this shouldn't change anything after ejabberd is built.